brain/task_executor.py

```
@@ -495,11 +496,53 @@ async def _assess_user_plugin(self, conversation: str, plugins: Any) -> UserPlug
+                # Try to fix common JSON issues before parsing
+                # Remove trailing commas before closing braces/brackets
+                # Fix trailing commas in objects and arrays
+                text = re.sub(r',(\s*[}\]])', r'\1', text)
+                # Remove any comments (though JSON doesn't support comments)
+                text = re.sub(r'//.*?$', '', text, flags=re.MULTILINE)
+                text = re.sub(r'/\*.*?\*/', '', text, flags=re.DOTALL)
+                
                 try:
                     decision = json.loads(text)
                 except Exception as e:
                     logger.exception(f"[UserPlugin Assessment] JSON parse error: {e}; raw_text (truncated): {repr(raw_text)[:2000]}")
```

Ruff (0.14.8)

Redundant exception object included in `logging.exception` call

(TRY401)